kubetest/e2e.go

```diff
@@ -762,18 +762,24 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, o.logexporterGCSPath, wg *sync.WaitGroup, provider, dump string) {
```

Comment:
It should be like:

```
func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump, logexporterGCSPath string) {
```

So fix the syntax and move logexporterGCSPath to the last position.
And then change kubemarkDown call (around ~249 line in this file) to something like:

```
		go kubemarkDown(&kubemarkDownErr,  &kubemarkWg, o.provider, dump, o.logexporterGCSPath)

```